### PR TITLE
awesome communities link added to navbar

### PR DESCRIPTION
**Pull request by tanyapowell**
| Section     | Details                              |
| ----------- | ------------------------------------ |
| Type        | Code  |
| Description | Adding the awesome communities URL to navbar  |
| Screenshot  | No                    |

### CODE REVIEW

File: _includes/navigation.html [hunk 1]
```diff
@@ -37,6 +37,11 @@
                     <span class="label label-pill label-info">{{ site.data.future-episodes.size }}</span>
                 </a>
             </li>
+             <li class="nav-item">
+                <a class="nav-link" href="http://werocktech.com/awesome-tech-communities/">
```
Comment (eddiejaoude) [inline _includes/navigation.html]: What do you think about opening in a new tab with using `target="_blank"`?

Comment (tanyapowell) [reply _includes/navigation.html]: yep, makes sense 👍 

Will also add to other external links and add to this PR 😄

Comment (eddiejaoude) [reply _includes/navigation.html]: Thank you 👍

File: _includes/navigation.html [hunk 2]
```diff
@@ -37,6 +37,11 @@
                     <span class="label label-pill label-info">{{ site.data.future-episodes.size }}</span>
                 </a>
             </li>
+             <li class="nav-item">
+                <a class="nav-link" href="http://werocktech.com/awesome-tech-communities/" target="_blank">
+                    <i class="fa fa-handshake-o" aria-hidden="true"></i> Communities
```
Comment (eddiejaoude) [inline _includes/navigation.html:42]: I noticed that this icon does not appear.

Comment (eddiejaoude) [reply _includes/navigation.html:42]: That icon is only available from font awesome `v4.7`

screenshot 2017-06-13 13 06 05

I will update **FontAwesome** as currently using `v4.5.0`

Comment (tanyapowell) [reply _includes/navigation.html:42]: ok, let me take it out and I'll find one that works when I get home 😄

Comment (tanyapowell) [reply _includes/navigation.html:42]: Oops! Didn't check the version of FontAwesome before adding (my bad!)

Comment (eddiejaoude) [reply _includes/navigation.html:42]: No worries, it happens to us all 😄 . Probably time to update though or switch to `cdn` so easier to update going forward 🤓

**Pull request merged**